src/main/java/mate/academy/controller/BookController.java

```diff
@@ -30,35 +37,86 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    @Operation(summary = "Get all books with pagination",
+            description = "Returns a paginated list of all books")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successfully retrieved list"),
+            @ApiResponse(responseCode = "400", description = "Invalid pagination parameters"),
+            @ApiResponse(responseCode = "500", description = "Internal server error")
+    })
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
Look at these annotations 
```suggestion
    public List<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {
```